src/iptax/ai/review.py

```
@@ -0,0 +1,596 @@
+}
+
+
+class ListScroll(VerticalScroll):
+    """VerticalScroll without up/down key bindings.
+    
+    This allows using up/down arrows for list navigation without
+    triggering scroll behavior. PageUp/Down and mouse scroll still work.
+    """
+
+    # Override to remove all navigation bindings - we handle them in the app
+    BINDINGS = []
```

Ruff (0.14.7)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)